lockin/src/trig.rs

```diff
@@ -0,0 +1,146 @@
+// `force_eval` is taken from libm/src/math/mod.rs.
+macro_rules! force_eval {
+    ($e:expr) => {
+        unsafe {
+            ::core::ptr::read_volatile(&$e);
+        }
+    };
+}
+
+#[path = "sinf.rs"]
+mod sinf;
+#[path = "k_sinf.rs"]
+mod k_sinf;
+#[path = "cosf.rs"]
+mod cosf;
+#[path = "k_cosf.rs"]
+mod k_cosf;
+#[path = "rem_pio2f.rs"]
+mod rem_pio2f;
```

Comment:
I think making a `trig/` folder with all of the routines in it would be more idiomatic and help to keep a clean directory structure as well.